src/main/resources/README.txt

```diff
@@ -68,53 +68,63 @@ The package includes:
 3. Release Notes
 ------------------------------------------------------------------------------
 
-The package includes the 1.10.0-rc1 version of the project.
+The package includes the 1.10.2 version of the project.
 This is a release candidate.
 
-### Changes in v1.10.0-rc1
-
-* Distribution
-  * [NEW] new graphical user interface (GUI)
-  * [NEW] installers for the GUI on Windows
-  * [NEW] packaged application on Mac OS X
-  * online Java installer bundled in the Windows installer
-  * Java Runtime Environment bundled in the Mac OS X app
-
-* Command Line Interface
-  * [NEW] `clean` command to remove jobs with an `ERROR` status
-  * inputs and options arguments are no longer prefixed with `--i` and `--x`
-  * the `version` command now works as expected
-  * Properly detect Java under OpenJDK and Ubuntu 15.04
-
-* Framework and API
-  * [NEW] Add datatypes to options and list them through the api
-  * [NEW] Expose the default value for options
-  * [NEW] Add support for job batches
-  * Get rid of the folder in zipped ports and options
-  * Catch out of memory errors
-  * Return a more meaningful error when inputs are not corect
-  * Improved control of script removal
-  * Improved logging filters
-  * Normalize whitespace in script documentation parsing
-  * Do not strip out spaces if xml:space="preserve"
-  * Allow posting job requests using a namespace prefix   
-
-* Modules
-  * [NEW] DAISY 2.02 validator
-  * [NEW] DAISY 3 (audio-only) to DAISY 2.02
-  * [NEW] EPUB 3 validator (EpubCheck)
-  * [NEW] EPUB 3 to PEF script
-  * [NEW] HTML to PEF script
-  * Experimental audio-only DAISY 3 production
-  * Improved MathML production in DAISY 3
-  * Improved NIMAS validation
-  * Optimized fileset lading
-  * ... and various other bug fixes
+### Installer/Updater
+
+- [FIX] Updater couldn't find the Pipeline installation from the registry on
+  64bit Windows
+- [FIX] Updater couldn't find releases info ("404 not found")
+
+### Framework and API
+
+- Support file names with spaces inside zipped job context
+- Improve search algorithm for binaries
+- [FIX] Spaces in paths of book files cause job failure
+
+### Modules
+
+- Add basic tests for all scripts
+- Re-enable all XSpec and and XProcSpec tests
+- daisy202-to-epub3
+  - [FIX] smil references inside links should also be removed
+  - [FIX] Whenever a `epub:textref` attribute is added to a SMIL, an
+    `attribute-value` attribute with the same value is added
+  - [FIX] `epub:textref` in SMIL refers to `.html` files instead of `.xhtml`
+    files
+  - [FIX] Remove superfluous xmlns:d from package document metadata
+- dtbook-to-epub3
+  - [FIX] Issue with whitespace being removed
+  - [FIX] The "assert validity" option on dtbook-to-epub3 does not seem to work
+- daisy202-validator
+  - [FIX] Attribute "shape" not allowed here
+- Braille modules

```

Comment:
More has happened since v1.10.0, but you probably didn't find all of the changes because not all of them have pull requests. By the way also things have happened between v1.10.0 and v1.9.16 of the braille modules which aren't in the release notes.